Userland/DevTools/GMLPlayground/MainWidget.cpp

```diff
@@ -69,7 +69,7 @@ ErrorOr<NonnullRefPtr<MainWidget>> MainWidget::try_create(GUI::Icon const& icon)
     main_widget->m_editor = main_widget->find_descendant_of_type_named<GUI::TextEditor>("text_editor");
     main_widget->m_preview_frame_widget = main_widget->find_descendant_of_type_named<GUI::Frame>("preview_frame");
 
-    main_widget->m_preview_window = TRY(GUI::Window::try_create());
+    main_widget->m_preview_window = TRY(GUI::Window::try_create(main_widget));
```

Comment:
Commit title is backwards, you're making the main widget the parent of the preview window. :^)